pom.xml

```diff
@@ -400,7 +400,7 @@ limitations under the License.
                         <exclude>.repository/**</exclude>
                         <exclude>**/target/**</exclude>
                         <exclude>data/*.txt</exclude>
-                        <exclude>bin/gremlin.sh</exclude>
+                        <exclude>**/bin/gremlin.sh</exclude>
```

Comment:
The file is not located in the root directory. I was not sure if the intent was to ignore any `gremlin.sh` or just the specific version. I assumed the former.

Reply:
former was correct i believe.